examples/lock-app/efr32/src/main.cpp

```diff
@@ -107,7 +107,7 @@ int main(void)
     efr32LogInit();
 #endif
 #endif
-
+    CHIPPlatformMemoryInit(0, 0);
```

Comment:
Is this change related to this PR ?

Reply:
This change is already present in master now. And it was necessary for EFR32 CI at the time.